src/Ole32/Ole32.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+    using System.Runtime.InteropServices;
+
+    /// <summary>
+    /// Exported functions from the Ole32.dll Windows library
+    /// that are available to Desktop and Store apps.
+    /// </summary>
+    [OfferFriendlyOverloads]
+    public static partial class Ole32
+    {
+        /// <summary>
+        /// Enables cancellation of synchronous calls on the calling thread.
+        /// </summary>
+        /// <param name="pReserved">This parameter is reserved and must be NULL.(<see cref="IntPtr.Zero"/>)</param>
+        /// <returns>This function can return the standard return values <see cref="HResult.Code.S_OK"/>, <see cref="HResult.Code.E_FAIL"/>, <see cref="HResult.Code.E_INVALIDARG"/>, and <see cref="HResult.Code.E_OUTOFMEMORY"/>.</returns>
+        [DllImport(nameof(Ole32))]
+        public static extern HResult CoEnableCallCancellation(IntPtr pReserved);
```

Comment:
I think strictly speaking this should be `void*` (and we'll automatically generate `IntPtr` overloads).